mlip_arena/models/externals/pet.py

```
@@ -0,0 +1,31 @@
+    def __init__(
+        self,
+        checkpoint=REGISTRY["PET(OAM)"]["checkpoint"],
+        device: str | None = None,
+        **kwargs,
+    ):
+        self.device = device or str(get_freer_device())
+        super().__init__(model="pet-oam-xl", version="1.0.0", device=self.device)
+    
+    
+    def calculate(
+        self, atoms=None, properties=['energy', 'forces', 'stress'], system_changes=all_changes
```

Ruff (0.14.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)